queries.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE teams
+    SET abbreviation 'WSH' = 'WAS'
```

Comment:
Nice attempt. Remember that when we want to target specific rows in our query we use a `WHERE` clause to do that.

`UPDATE teams SET abbreviation = 'WAS' WHERE abbreviation = 'WSH';`

This will start by looking for rows that have an abbreviation that is `WSH` and when it finds them (in this case only the one) it will update the `abbreviation` column for to be `WAS` instead.